packages/ui/src/components/Button/index.tsx

```diff
@@ -9,7 +9,9 @@ export interface ButtonProps {
   className?: string;
   onClick: (event: React.MouseEvent<HTMLButtonElement>) => void;
   children: React.ReactNode;
-  size?: 'small' | 'medium';
+  size?: 'small' | 'medium' | 'large' | '';
+  full?: boolean;
```

Comment:
Perhaps we call this
```suggestion
  fullWidth?: boolean;
```